tox.ini

```diff
@@ -71,14 +71,14 @@ allowlist_externals =
 find = git ls-files
 filter = file --mime-type -Nnf- | grep shellscript | cut -f1 -d:
 
-[testenv:lint-{black,ruff,shellcheck,codespell,yaml}]
+[testenv:lint-{ruff,shellcheck,codespell,yaml}]
 description = Lint the source code
 base = testenv, lint
 labels = lint
 commands_pre =
     shellcheck: bash -c '{[shellcheck]find} | {[shellcheck]filter} > {env_tmp_dir}/shellcheck_files'
 commands =
-    black: black --check --diff {tty:--color} {posargs} .
+    ruff: ruff format --diff {posargs:.}
```

Comment:
how does this work with two `ruff: ` entries?

Reply:
It runs the entries in definition order.